### PR TITLE
GEOMESA-1522 Add geomesa-site.xml.template to hbase/bigtable distros

### PR DESCRIPTION
**Pull request by aheyne**
Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
need it in geomesa-hbase-dist too I think?

**Comment by aheyne** [comment]
derp, fixed.

File: geomesa-hbase/geomesa-hbase-dist/src/main/assemblies/component.xml [hunk 1]
```diff
@@ -106,6 +106,12 @@
             <destName>README-convert.md</destName>
             <outputDirectory>docs</outputDirectory>
         </file>
+        <!-- geomesa-site.xml.template -->
+        <file>
+            <source>../../../geomesa-utils/src/main/resources/org/locationtech/geomesa/geomesa-site.xml.template</source>
```
Comment (elahrvivaz) [inline geomesa-hbase/geomesa-hbase-dist/src/main/assemblies/component.xml]: these paths are wrong - need to be 2 levels up not 3

Comment (aheyne) [inline geomesa-hbase/geomesa-hbase-dist/src/main/assemblies/component.xml]: dot walking should be a sin, fixed.

Comment (elahrvivaz) [inline geomesa-hbase/geomesa-hbase-dist/src/main/assemblies/component.xml]: agreed, but i think it's the only thing maven supports that still works if you don't build from the top-level dir...

**Pull request merged**